### PR TITLE
Add node selectors to many components.

### PR DESCRIPTION
**Pull request by PFedak**

### CODE REVIEW

File: etc/helm/pachyderm/values.yaml [hunk 1]
```diff
@@ -503,6 +507,8 @@ postgresql:
     size: 10Gi
     labels:
       suite: pachyderm
+  primary:
+    nodeSelector: {}
```
Comment (PFedak) [inline etc/helm/pachyderm/values.yaml:511]: I'm not clear if this needs to be here, or should be here by convention, as this is just passing the value through to the subchart

**Comment by acohen4** [comment]
could you also add values for tolerations (same way as nodeSelector). I have an example PR here: https://github.com/pachyderm/pachyderm/pull/7177/files

Also, could you run `make schema` from ./etc/helm ? It adds some type-checking to our helm templates

File: etc/helm/pachyderm/values.yaml (hunk 1 above)
Comment (acohen4) [reply etc/helm/pachyderm/values.yaml:511]: where'd "primary" come from lol?

Comment (PFedak) [reply etc/helm/pachyderm/values.yaml:511]: ah, maybe I'm doing it wrong. It seemed like the pod-related stuff in the postegresql chart was under `primary:`, maybe I don't understand how that works

Comment (acohen4) [reply etc/helm/pachyderm/values.yaml:515]: so my bad - you're totally right here. It's how the postgresql subchart distinguishes between config for the primary write PG replica vs the read replicas that can be deployed https://github.com/pachyderm/pachyderm/blob/master/etc/helm/pachyderm/charts/postgresql/templates/statefulset-readreplicas.yaml. Could you also add a similar readReplicas section?

**Review by acohen4 — APPROVED** [review]
LGTM!

**Review by tybritten — APPROVED** [review]
same, looks good.

File: etc/helm/pachyderm/values.yaml (hunk 1 above)
Comment (PFedak) [reply etc/helm/pachyderm/values.yaml:515]: ah, nice catch, that naming makes more sense now

**Pull request merged**